homeassistant/components/litterrobot/vacuum.py

```diff
@@ -136,6 +146,13 @@ async def async_set_sleep_mode(
 
     async def async_set_wait_time(self, minutes: int) -> None:
         """Set the wait time."""
+        # The Litter-Robot set wait time service has been replaced by a
+        # dedicated select entity and marked as deprecated
+        _LOGGER.warning(
+            "The 'litterrobot.set_wait_time' service is deprecated and "
+            "replaced by a dedicated set wait time select entity; Please "
+            "use that entity to set the wait time instead"
+        )
```

Comment:
This seems incorrect? I only see the reset waste drawer button in this PR?

Reply:
This was introduced in https://github.com/home-assistant/core/pull/58323, I figured since I was marking the other service as deprecated due to a new entity, it should also be done for this one. I can put this in a different PR if needed.